Yosemite/Yosemite/Actions/ProductVariationAction.swift

```diff
@@ -10,6 +10,10 @@ public enum ProductVariationAction: Action {
     ///
     case synchronizeProductVariations(siteID: Int64, productID: Int64, pageNumber: Int, pageSize: Int, onCompletion: (Error?) -> Void)
 
+    /// Retrieves the specified ProductVariation.
+    ///
+    case retrieveProductVariation(siteID: Int64, productID: Int64, variationID: Int64, onCompletion: (Result<ProductVariation, ProductUpdateError>) -> Void)
```

Comment:
I think it'd be better for the completion block to return a generic `Error` instead of `ProductUpdateError`, since this action isn't related to saving/updating a product